code/src/components/Nav.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom';
+
+import '../styles/Nav.css'
+import MovieIcon from 'icon/icon.png'
```

Comment:
Uppercase names are reserved for components only, not images. Should rather be :

```
import movieIcon from 'icon/icon.png'
```